ts/Data/DataParser.ts

```diff
@@ -0,0 +1,45 @@
+/* *
+ *
+ *  Data module
+ *
+ *  (c) 2012-2020 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import DataTable from './DataTable.js';
+import DataRow from './DataRow.js';
+import U from '../Core/Utilities.js';
+const {
+    uniqueKey
+} = U;
+
+class DataParser {
+
+    /**
+     * Functions
+     */
+
+    public columnArrayToDataTable(columns: Array<Array<Highcharts.DataValueType>>, headers: string[] = []): DataTable {
```

Comment:
Referencing the Highcharts namespace (which exists only for migration) makes it not a real ES6 module.